app/models/post.rb

```diff
@@ -0,0 +1,32 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :comments
+  has_many :likes
+
+  after_save :update_posts_counter
+
+  def update_posts_counter
+    author.increment!(:posts_counter)
+  end
+
```

Comment:
### [ OPTIONAL]
- [ ] Great work using the `after_save` method to call your function 👏🏿 . However, make sure that your function is private when you use the method. This will make sure that your function can not be used anywhere else in your application

Reply:
Thank you Peterrobert, 🙏🏻  I'm going to add a `private` property before the update_posts_counter method so as to make it private